src/files/examples/fingerprint_reader_functions.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python
+
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#
+#       DESCRIPTION: Having an example all in one file including miscancellous possibilites of the 
+#		library and being able to call them from command line
+#
+#       CALL SAMPLE:
+#               Usage  python2 fingerprint_reader_function.py -h will give you the possibilities
+#
+#       @author: Philippe Gachoud inspired from Bastian Raschke files and examples
+#       @creation: 20180101
+#       @last modification:
+#       @version: 1.0
+#       @URL: $URL
+#
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+################################################################################
+# SCRIPT_DIR SETTING
+################################################################################
+
+################################################################################
+################################################################################
+# INCLUDES
+################################################################################
+################################################################################
+import sys
+import logging # http://www.onlamp.com/pub/a/python/2005/06/02/logging.html
+import time
+from pyfingerprint.pyfingerprint import PyFingerprint
+import argparse
+import hashlib
+
+################################################################################
+################################################################################
+# CONSTANTS
+################################################################################
+################################################################################
+
+################################################################################
+################################################################################
+# VARIABLES
+################################################################################ 
+################################################################################ 
+
+global f # Fingerprintreader
+
+################################################################################
+################################################################################
+# FUNCTIONS DEFINITION 
+################################################################################
+################################################################################
+def init():
+	global f
+	logging.basicConfig(level=logging.DEBUG, stream=sys.stdout, filemode="a+", format="%(asctime)-15s %(levelname)-8s %(message)s")
+	try:
+	    f = PyFingerprint('/dev/ttyUSB0', 57600, 0xFFFFFFFF, 0x00000000)
```

Comment:
Why not pass via argument (like other options below) e.g. `--device=/dev/ttyUSB0`

Reply:
why not...